htdocs/societe/card.php

```diff
@@ -2910,15 +2920,21 @@ function init_supplier_categ() {
 				if (empty($user->socid)) {
 					if (!empty($object->email) || $at_least_one_email_contact) {
 						$langs->load("mails");
-						print '<a class="butAction" href="'.$_SERVER['PHP_SELF'].'?socid='.$object->id.'&action=presend&mode=init#formmailbeforetitle">'.$langs->trans('SendMail').'</a>'."\n";
+						$params = array(
+							'attr' => array(
+								'title' => ''
+							)
+						);
```

Comment:
If $param is empty don't set it and leave default value

Reply:
add on function dolGetButtonAction